src/database/factories/user.factory.ts

```diff
@@ -5,12 +5,14 @@ import { User } from '@entities/user.entity';
 define(User, (faker: typeof Faker) => {
   const firstName = faker.name.firstName();
   const lastName = faker.name.lastName();
+  const gender = faker.random.arrayElement(['male', 'female', 'nonbinary']);
```

Comment:
What do you think about using an Enum to store these values. So you can avoid breaking the DRY principle.

Reply:
Great suggestion, I updated it and did it with Enum in the new commit.